### ISSUE TITLE
First page of web installer is confusing - says 'Error' wrongly

### ISSUE THREAD

**Issue by John-S4** [boxbilling/boxbilling#937]
### What is the idea?

The suggested way in all of the documentation to install BB is to upload the latest released files, unzip, and then run the web installer. 

When you do that, the first thing that you get is a page that has a big word **Error** at the top, followed by telling you that your configuration file is invalid, and that you haven't updated it. 

Only after all of that do you get to the part that tells you can continue to use the web based installer. 

This seems like completely backwards behaviour. Someone has followed the installation instructions exactly and at the first point they get told there is an error (which isn't actually an error at all). 

I propose rewriting this page to say something more like:

_**Welcome**

Then link to continue to web based installer. 

Then information that alternatively you can edit the config file directly if you want to._ 

### Why would this be beneficial

- Removes an error message that isn't actually an error at all. 
- Makes things much easier to understand for first time user. 
- Aligns the installer and the documentation much better.

**Comment by evrifaessa** [comment]
Good point. Will be looking forward to a PR.

**Comment by BenNottelling** [comment]
@John-S4 When you work on this, please also remove the reference to the Facebook
https://github.com/boxbilling/boxbilling/blob/master/src/install/assets/install.phtml#L43
Also, it would be nice to get some of these link duplications taken care of in the CSS
https://sonarcloud.io/code?id=BenNottelling_boxbilling&selected=BenNottelling_boxbilling%3Asrc%2Finstall%2Fassets%2Fcss%2Fmain.css

Thanks :)

**Comment by John-S4** [comment]
Yep, I'll remove the FB reference and also the links to the dead forums on boxbilling.com
